src/rosdep2/platforms/gentoo.py

```diff
@@ -54,10 +58,21 @@ def register_installers(context):
     context.set_installer(PORTAGE_INSTALLER, PortageInstaller())
 
 def register_platforms(context):
+    register_funtoo(context)
+    register_gentoo(context)
+
+def register_gentoo(context):
     context.add_os_installer_key(OS_GENTOO, PORTAGE_INSTALLER)
     context.add_os_installer_key(OS_GENTOO, SOURCE_INSTALLER)
     context.set_default_os_installer_key(OS_GENTOO, PORTAGE_INSTALLER)
 
+def register_funtoo(context):
```

Comment:
I think I was trying to kill two issues with one PR here...

Not even sure where this code is, anymore (has it _really_ been two years??).